pkg/apis/defaults/defaults.go

```diff
@@ -44,6 +44,7 @@ func SetDefaultAttributes(attrOriginal map[string]string) (map[string]string, er
 	setDefaultIfEmpty(attr, csiapi.KeyFileKey, "tls.key")
 
 	setDefaultIfEmpty(attr, csiapi.KeyEncodingKey, "PKCS1")
+	setDefaultIfEmpty(attr, csiapi.KeystoreType, "PKCS12")
```

Comment:
Not against it, but do we want to have a default value for this, or would we prefer users to explicitly specify this? Do we think PKCS12 is the more common one and makes sense to default?